### DISCUSSION TITLE
hostloc主机论坛 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#25] [模板讨论]
hostloc主机论坛 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by 18690819111** [comment]
Failed at 2/8 request,
Fail assert: {"re": "自动登录", "from": "content"} from failed_asserts,
Request URL: https://hostloc.com/forum.php

**Reply by wjf0214** [reply]
这个提示是cookie失效。
如果确认自己复制的没问题。请尝试右键选中后复制，不要直接右键复制值。

**Reply by 18690819111** [reply]
谢谢我试一下

**Comment by ipme** [comment]
Failed at 1/8 request,
Fail assert: {"re": "\"状态\": \"200\"", "from": "content"} from success_asserts,
Response Error : HTTP 500: The error occurred when rendering template data: content={{random(0,1,n)}}
TypeError('Random.choice() takes 2 positional arguments but 4 were given'),
Request URL: api://util/unicode

**Comment by hecarli555** [comment]
请问N值最大可以设置多少？不会被站点抓的吧？

**Reply by wjf0214** [reply]
这个没限制，只是太多也没必要，也没意义。只是为了完成访问空间的任务。

**Comment by SayXieXie** [comment]
这个签到需要代理才能成功吗？

**Reply by wjf0214** [reply]
你要直连可以请求成功，那就不需要。反正就需要。。。

**Comment by hecarli555** [comment]
这几天报错了。。。这啥问题呢？
2024-11-04 15:39:49
日志：Failed at 2/8 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: quictls/3.1.5: error:0A000438:SSL routines::tlsv1 alert internal error,
Request URL: https://hostloc.com/forum.php

**Comment by hecarli555** [comment]
<html>
<body>

Failed at 2/8 request,Fail assert: {"re": "200", "from": "status"} from success_asserts,Response Error : HTTP 418: I'm a Teapot,Request URL: https://hostloc.com/forum.php
--



</body>
</html>请问大佬怎么破？
